script/foreman-debug

```diff
@@ -277,7 +277,12 @@ install_clean_trap() {
 if [ -z "$DIR" ]; then
   DIR=$(mktemp -d $(basename $0)-XXXXX -p /var/tmp)
 else
-  [ ! -d "$DIR" ] && mkdir -p "$DIR"
+  if [ -d "$DIR" ]; then
+    echo "Directory must not exist: $DIR"
+    exit 1
+  else
+    mkdir -p "$DIR"
+  fi
```

Comment:
This is actually not needed too for the tailing slash fix. It works even if you provide existing directory, it correctly only deletes it tho only if it's in `/var/tmp`.

```
[root@next ~]# mkdir /tmp/test3
[root@next ~]# foreman-debug -g -d /tmp/test3
Processing... (takes a while)

 HOSTNAME: next.nat.lan
       OS: redhat
  RELEASE: Red Hat Enterprise Linux Server release 7.5 (Maipo)
  FOREMAN: 1.18.0.14
     RUBY: ruby 2.0.0p648 (2015-12-16) [x86_64-linux]
   PUPPET: 5.5.0
  DENIALS: 0


A debug file has been created: /tmp/test3.tar.xz (537156 bytes)

To upload a tarball to our secure site, please use the -u option.
```

Just fix the trailing slash please, thanks.